sig/environment_helpers.rbs

```diff
@@ -0,0 +1,96 @@
+module EnvironmentHelpers
+  VERSION: String
+
+  class Error < StandardError
+  end
+
+  class MissingVariableError < Error
+  end
+
+  class BadDefault < Error
+  end
+
+  class BadFormat < Error
+  end
+
+  class InvalidType < Error
+  end
+
+  class InvalidValue < Error
+  end
+
+  class InvalidBooleanText < InvalidValue
+  end
+
+  class InvalidRangeText < InvalidValue
+  end
+
+  class InvalidIntegerText < InvalidValue
+  end
+
+  class InvalidDateText < InvalidValue
+  end
+
+  class InvalidDateTimeText < InvalidValue
+  end
+
+  module AccessHelpers
+    # (Actually provided by ENV, which these are all extended onto)
+    def fetch: (String name) -> String?
```

Comment:
You may want to use an optional parameter, as `(String name, ?String? default) -> String?`. But they are equivalent in this case.

Reply:
Yes, that's definitely more appropriate, thanks :-)